db/migrate/20210513132303_create_relationshop_between_article_and_user.rb

```diff
@@ -0,0 +1,5 @@
+class CreateRelationshopBetweenArticleAndUser < ActiveRecord::Migration[6.1]
```

Comment:
Not sure if it can be changed without causing issues, but Relationshop might be confusing later.

Reply:
Good eye 👁️